docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -260,49 +253,36 @@ cd synthetics/examples/todos/ &&\
 npm install
 ----
 
-You are now inside the a synthetics test-suite, which is also an NPM project.
-You can now run the provided tests; Note that, by default, only files matching the filename `*.journey.(ts|js)*`
-will be run.
+You are now inside the synthetics test suite, which is also an NPM project.
+You can now run the provided tests.
+By default only files matching the filename `*.journey.(ts|js)*` will be run.
 
 [source,sh]
 ----
-# Run tests on the current directory. Please note the dot `.` which indicates 
-# that we want to run tests on the current directory
+# Run tests on the current directory. The dot `.` indicates
+# that it should run all tests in the current directory.
 npx @elastic/synthetics .
 ----
 
-Once you have your tests up and running, follow the steps in the <<synthetics-quickstart,quickstart guide>>
-to integrate with the provided Docker project template.
-
-// Results. . .
-// [source,sh]
-// ----
-// Journey: basic addition and completion of single task
-//    ✓  Step: 'go to app' succeeded (150 ms)
-//    ✓  Step: 'add task Dont put salt in your eyes' succeeded (79 ms)
-//    ✓  Step: 'check that task list has exactly 1 elements' succeeded (7 ms)
-//    ✓  Step: 'check for task 'Don't put salt in your eyes' in list' succeeded (50 ms)
-//    ✓  Step: 'destroy task 'Don't put salt in your eyes'' succeeded (75 ms)
-//    ✓  Step: 'check that task list has exactly 0 elements' succeeded (2 ms)
-
-// Journey: adding and removing a few tasks
-//    ✓  Step: 'go to app' succeeded (125 ms)
-//    ✓  Step: 'add task Task 1' succeeded (40 ms)
-//    ✓  Step: 'add task Task 2' succeeded (49 ms)
-//    ✓  Step: 'add task Task 3' succeeded (21 ms)
-//    ✓  Step: 'check that task list has exactly 3 elements' succeeded (8 ms)
-//    ✓  Step: 'destroy task 'Task 2'' succeeded (94 ms)
-//    ✓  Step: 'check that task list has exactly 2 elements' succeeded (5 ms)
-//    ✓  Step: 'add task Task 4' succeeded (36 ms)
-//    ✓  Step: 'check that task list has exactly 3 elements' succeeded (4 ms)
-
-// Journey: check that title is present
-//    ✓  Step: 'go to app' succeeded (139 ms)
-//    ✓  Step: 'check title is present' succeeded (27 ms)
-
-// Journey: check that input placeholder is correct
-//    ✓  Step: 'go to app' succeeded (121 ms)
-//    ✓  Step: 'check title is present' succeeded (18 ms)
-
-//  19 passed (2983 ms)
-// ----
+After testing locally, you can run the test suite using the
+Elastic Synthetics integration configuration or `heartbeat.yml`.
```

Comment:
Nit: maybe `Elastic Synthetics integration` would sound better just for now while we don't have a "node integration"?

```suggestion
Elastic Synthetics integration or `heartbeat.yml`.
```